drivers/hmis/app/models/hmis/hud/processors/base.rb

```diff
@@ -106,7 +106,7 @@ def attribute_value_for_enum(enum_type, value)
     elsif is_array
       value.map { |val| enum_type&.value_for(val) || val }
     else
-      enum_type&.value_for(value) || value
+      enum_type&.flexible_value_for(value) || value
```

Comment:
I think to reduce impact, we can drop this now since we agreed on using the simpler radio buttons instead.. I'm sorry for all the back and forth @martha